sklearn/utils/metaestimators.py

```diff
@@ -32,7 +52,11 @@ def __get__(self, obj, type=None):
         if obj is not None:
             # delegate only on instances, not the classes.
             # this is to allow access to the docstrings.
-            self.get_attribute(obj)
+            for item in self.attribute_name:
+                if _hasattr_nested(obj, item):
+                    break
+            else:
+                attrgetter(self.attribute_name[-1])(obj)
```

Comment:
No, because you're looking up `best_estimator_.predict`, for instance, and if `best_estimator_` exists but its `predict` does not, you're backing off to `estimator`, when you should be raising an `AttributeError`.